remoting/subsystem/src/main/resources/org/jboss/as/remoting/LocalDescriptions.properties

```diff
@@ -7,13 +7,13 @@ remoting.worker-task-core-threads=The number of core threads for the remoting wo
 remoting.worker-task-core-threads.deprecated=Use IO subsystem worker configuration
 remoting.worker-task-keepalive=The number of milliseconds to keep non-core remoting worker task threads alive.
 remoting.worker-task-keepalive.deprecated=Use IO subsystem worker configuration
-remoting.worker-task-limit=The maximum number of remoting worker tasks to allow before rejecting.
+remoting.worker-task-limit=The maximum number of remote worker tasks to allow before rejecting them.
```

Comment:
Unless @fl4via disagrees, I think 'remoting' is correct. It's not an adjective describing the compound noun 'worker tasks', such that the tasks are 'remote' (from what?). It's a 3rd noun in a compound noun, 'remoting work tasks', with 'remoting' meaning JBoss Remoting.

Perhaps it should be more clear and say 'JBoss Remoting' or perhaps 'remoting' should just be dropped.